packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -65,6 +66,12 @@ export type CallCompositeOptions = {
    * @defaultValue true
    */
   callControls?: boolean | CallControlOptions;
+  /**
+   * Customize video rendering options
+   * Can be customized by providing an object of type {@link @azure/communication-react#VideoRenderingControlOptions}.
+   * @beta
+   */
+  videoRenderingControls?: VideoRenderingControlOptions;
```

Comment:
for composites we dictate with strong opinion what UI is shown to the users, so this customization should be removed at the composite level and just always be enabled

Reply:
So should we just remove videoRenderingControls for now and just always use loading spinner for remote video streams? Tbh it should always be enabled anyways

Reply:
We have decided to just remove videoRenderingControlOptions